src/deepsparse/open_pif_paf/annotate.py

```diff
@@ -0,0 +1,233 @@
+    required=True,
+    help="File path to image or directory of .jpg files, a .mp4 video, "
+    "or an integer (i.e. 0) for webcam",
+)
+@click.option(
+    "--engine",
+    type=click.Choice([DEEPSPARSE_ENGINE, ORT_ENGINE, TORCH_ENGINE]),
+    default=DEEPSPARSE_ENGINE,
+    help="Inference engine backend to run on. Choices are 'deepsparse', "
+    "'onnxruntime', and 'torch'. Default is 'deepsparse'",
+)
+@click.option(
+    "--image_shape",
+    "--image-shape",
+    type=int,
+    nargs=2,
+    default=(384, 384),
+    help="Image shape to use for inference, must be two integers",
+    show_default=True,
+)
+@click.option(
+    "--num_cores",
+    "--num-cores",
+    type=int,
+    default=None,
+    help="The number of physical cores to run the annotations with, "
+    "defaults to using all physical cores available on the system."
+    " For DeepSparse benchmarks, this value is the number of cores "
+    "per socket",
+    show_default=True,
+)
+@click.option(
+    "--save_dir",
+    "--save-dir",
+    type=click.Path(dir_okay=True, file_okay=False),
+    default="annotation-results",
+    callback=create_dir_callback,
+    help="The path to the directory for saving results",
+    show_default=True,
+)
+@click.option(
+    "--name",
+    type=str,
+    default=None,
+    help="Name of directory in save-dir to write results to. defaults to "
+    "{engine}-annotations-{run_number}",
+)
+@click.option(
+    "--target_fps",
+    "--target-fps",
+    type=float,
+    default=None,
+    help="Target FPS when writing video files. Frames will be dropped to "
+    "closely match target FPS. --source must be a video file and if "
+    "target-fps is greater than the source video fps then it "
+    "will be ignored",
+    show_default=True,
+)
+@click.option(
+    "--no_save",
+    "--no-save",
+    is_flag=True,
+    help="Set flag when source is from webcam to not save results."
+    "Not supported for non-webcam sources",
+    show_default=True,
+)
+def main(
+    model_filepath: str,
+    source: str,
+    engine: str,
+    image_shape: Tuple[int, int],
+    num_cores: Optional[int],
+    save_dir: str,
+    name: Optional[str],
+    target_fps: Optional[float],
+    no_save: bool,
+) -> None:
+    """
+    Annotation Script for OpenPifPaf with DeepSparse
+    """
+    save_dir = get_annotations_save_dir(
+        initial_save_dir=save_dir,
+        tag=name,
+        engine=engine,
+    )
+
+    loader, saver, is_video = get_image_loader_and_saver(
+        path=source,
+        save_dir=save_dir,
+        image_shape=image_shape,
+        target_fps=target_fps,
+        no_save=no_save,
+    )
+
+    is_webcam = source.isnumeric()
+    open_pif_paf_pipeline = Pipeline.create(
+        task="open_pif_paf",
+        model_path=model_filepath,
+        engine_type=engine,
+        num_cores=num_cores,
+    )
+
+    for iteration, (input_image, source_image) in enumerate(loader):
+        if iteration % 5 != 0:
+            continue
```

Comment:
optional nit: make this a cli arg?

Reply:
This should not be there. A small code change to create demos for Alex faster :)